packages/suite/src/constants/suite/urls.ts

```diff
@@ -4,6 +4,8 @@ export const CHANGELOG_MARKDOWN_URL =
 export const TREZOR_URL = 'https://trezor.io';
 export const TREZOR_COINS_URL = 'https://trezor.io/coins/';
 export const SUPPORT_URL = 'https://trezor.io/support/';
+export const CANNOT_ACCESS_WALLET =
+    'https://trezor.io/support/technical/i-cannot-access-the-wallet/';
```

Comment:
```suggestion
export const WALLET_ACCESS_URL =
    'https://trezor.io/support/technical/i-cannot-access-the-wallet/';
```
All other variables have `URL` in its name. I would keep it consistent